mod/developers/start.php

```diff
@@ -190,7 +191,8 @@ function developers_wrap_views($hook, $type, $result, $params) {
 	}
 
 	if ($result) {
-		$result = "<!-- developers:begin $view -->$result<!-- developers:end $view -->";
+		$filename = _elgg_services()->views->findViewFile($view, 'default');
```

Comment:
In that case it should also include all hooks that are modifying the view. I just want something simple that tells me which plugin is messing with my presentation.